### PR TITLE
[Misc] Add Mozilla cacerts

### PR DESCRIPTION
**Pull request by yfxhust**
Summary: Add mozilla cacerts

Test Plan: Covered by Mozilla Network Security Service

Reviewed-by: D-D-H, luchsh

Issue: https://github.com/alibaba/dragonwell8/issues/29

### CODE REVIEW

**Comment by luchsh** [comment]
1, pls sign CLA, anyway
2, let's move this file to some more suitable directory,
not the root directory :(

**Comment by yfxhust** [comment]
> 1, pls sign CLA, anyway
> 2, let's move this file to some more suitable directory,
> not the root directory :(

Accept. Already move cacerts file to common/security. I also add another cacerts.list.txt to make cacerts human readable.

**Comment by luchsh** [comment]
Please create an issue, link it to this patch, and update the 'Issue: ' field from commit template.

**Comment by luchsh** [comment]
> Test Plan: Covered by Mozilla Network Security Service
That does sound sufficient to me, is there any existing JTreg/JCK tests related to this change?

**Comment by luchsh** [comment]
> > 1, pls sign CLA, anyway
> > 2, let's move this file to some more suitable directory,
> > not the root directory :(
> 
> Accept. Already move cacerts file to common/security. I also add another cacerts.list.txt to make cacerts human readable.

how was `cacerts.list.txt` created? is it just for human to read?
pls add copyright header to that file if possible.

**Comment by yfxhust** [comment]
Make some improvement according to luchsh comments.
1. add Isssue field in comment
2. add copyright in cacerts.list.txt
3. This cacerts already verified by Mozilla nss project. Introducing nss test is not neccessary.

**Comment by D-D-H** [comment]
>                  --with-cacerts-file=`pwd`/common/security/cacerts \
pwd is not robust, how about readlink -f ./

File: make.sh [hunk 1]
```diff
@@ -62,12 +60,17 @@ else
   BUILD_INDEX=b$BUILD_NUMBER
 fi
 
+DISTRO_VERSION=${DRAGONWELL_VERSION}-${BUILD_INDEX}
+DISTRO_NAME="Alibaba Dragonwell"
+
```
Comment (luchsh) [inline make.sh]: pls move this change to another patch。
and you should update common/autoconf/spec.gmk.in instead

File: dragonwell_version [hunk 2]
```diff
@@ -19,5 +19,6 @@
 # Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
 #
 
-DRAGONWELL_VERSION=8.0-preview
+DRAGONWELL_VERSION=8.0.0
 DRAGONWELL_JDK_UPDATE_VERSION=202
+DRAGONWELL_JDK_BUILD_NUMBER=b01
```
Comment (luchsh) [inline dragonwell_version]: should not change this, pls create another patch
